### PR TITLE
Update readme

### LINKED ISSUE #46: Update README
## Is your feature request related to a problem? Please describe.
The current README is very out of date and several sections are completely blank.

## Describe the solution you'd like
Update the README to fill in as many sections as possible!

### PR DESCRIPTION
**Pull request by p-galligan**
Updates readme to fill in as much as possible. Routes are incomplete as of now, but will be extended as necessary. Fixes #46

### CODE REVIEW

**Review by bonniegee — APPROVED** [review]

**Comment by p-galligan** [comment]
Sorry @helrond @HaSistrunk I thought I asked for your review on this as well. Adding you now.

File: README.md [hunk 1]
```diff
@@ -26,13 +28,23 @@ Or, if you want to remove all data
 
 ## Services
 
+* Iterates over a list of request URIs and gets corresponding data from ArchivesSpace and parses over the data and marks it as submittable or unsubmittable.
```
Comment (helrond) [inline README.md]:
```suggestion
* Request Pre-Processing: Iterates over a list of request URIs, fetches corresponding data from ArchivesSpace, parses the data and marks it as submittable or unsubmittable.
```

File: README.md [hunk 2]
```diff
@@ -26,13 +28,23 @@ Or, if you want to remove all data
...
+* Converts AS data into strings and it to the message body of an email and then sends an email to an address or list of addresses.
```
Comment (helrond) [inline README.md]:
```suggestion
* Mailer: correctly formats the body of an email message and sends an email to an address or list of addresses.
```

File: README.md [hunk 3]
```diff
@@ -26,13 +28,23 @@ Or, if you want to remove all data
...
+* Creates retrieval and duplication transactions in Aeon by sending data to the Aeon API.
```
Comment (helrond) [inline README.md]:
```suggestion
* Aeon Request Submission: creates retrieval and duplication transactions in Aeon by sending data to the Aeon API.
```

File: README.md [hunk 4]
```diff
@@ -26,13 +28,23 @@ Or, if you want to remove all data
 
 ## Services
```
Comment (helrond) [inline README.md:29]: There's also the CSV Download service that's missing.

**Review by helrond — CHANGES_REQUESTED** [review]

**Review by helrond — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 # Request Broker
 
-An application to process retrieval and duplication requests from our discovery system, and route relevant item data to our retrieval management system and back to researchers.
+An application that accepts requests with lists of ArchivesSpace URIs from users of our discovery system and for each item grabs data from ArchivesSpace. Formats that data for delivery to our item retrieval system (Aeon) for retrieval or digital duplication, or formats the data for email delivery or CSV download. Routes formatted data to retrieval management system or to an email for researcher use.
```
Comment (HaSistrunk) [inline README.md]:
```suggestion
An application that accepts requests with lists of ArchivesSpace URIs from users of our discovery system. For each item, the application fetches data from ArchivesSpace, formats that data for delivery to our retrieval management system (Aeon) to enable reading room or duplication requests, or formats the data for email delivery or CSV download. It routes formatted data to the retrieval system or to an email for researcher use. This is a passthrough service; users cannot create requests directly in the application.
```

**Comment by HaSistrunk** [comment]
Looks good - I just had a few suggestions to make that first paragraph a little clearer.

**Review by HaSistrunk — APPROVED** [review]

**Pull request merged**